app/assets/css/sass/_popup.scss

```diff
@@ -459,6 +463,15 @@
 	p {
 		margin: 0;
 	}
+	
+	.rtmedia-comment-media-container{
+		margin:0;
+	}
+
+	.rtmedia-comment{
```

Comment:
Keep all code in a consistent way, add one space before starting bracket at all places.